packages/kit/src/core/node/index.js

```diff
@@ -1,15 +1,13 @@
-import { isContentTypeTextual } from '../adapter-utils.js';
-
 /**
  * @param {import('http').IncomingMessage} req
- * @returns {Promise<import('types/hooks').StrictBody>}
+ * @returns {Promise<Uint8Array>}
  */
 export function getRawBody(req) {
 	return new Promise((fulfil, reject) => {
 		const h = req.headers;
 
 		if (!h['content-type']) {
-			return fulfil('');
+			return fulfil(new Uint8Array());
```

Comment:
I think it's better to pass empty arrays instead of `null` for the following reasons:
- Typescript tells the user that `rawBody` will always be an `Uint8Array`.
- It requires an explicit cast to `any` which I consider unsafe (`/** @type {any} */ (null)`).
- It avoids needing to check if `rawBody` is `null` on other parts of the code

Reply:
Ah, of course — `strictNullChecks`. I do feel like `null` is a more correct representation of 'no body was provided' than an empty array. It also means we're not unnecessarily creating those typed arrays — the cost may be negligible, but I'd argue it's almost always better to avoid doing extra work at runtime for the sake of neater types. Tried it in 86c3d94409b78b791b7fe5fb6d8ea927726751d4, the tests still pass etc

Reply:
Looks good to me, but I have one more question, should we update the `BaseBody` type to include `null`?

Reply:
oh, right — yeah, we probably should